prow/cmd/deck/main.go

```diff
@@ -79,6 +84,15 @@ type options struct {
 }
 
 func (o *options) Validate() error {
+	if o.buildCluster != "" {
+		logrus.Error("--build-cluster is deprecated, switch to --cluster by June 2019.")
```

Comment:
Endlessly frustrating that we use different flag names ...

Reply:
Have you tried to use git and/or anything else in linux? :smiling_imp: